src/native/managed/cdacreader/src/DataType.cs

```diff
@@ -23,6 +23,7 @@ public enum DataType
     Thread,
     ThreadStore,
     GCAllocContext,
+    EEAllocContext,
```

Comment:
@elinor-fung @lambdageek Please review the contract additions

Reply:
ping @elinor-fung @lambdageek - any concerns here? This is the only outstanding part of the code review remaining.